tests/Microsoft.Fx.Portability.Tests/Analysis/AnalysisEngine.NuGetPackageInfo.Tests.cs

```diff
@@ -176,17 +176,46 @@ public void ComputeAssembliesToRemove_AssemblyExplicitlyPassedIn()
             Assert.Empty(assemblies);
         }
 
-        private static AssemblyInfo GetAssemblyInfo(string assemblyName, string version, bool isExplicitlySpecified)
+        /// <summary>
+        /// Tests that if the flag isExplicitlySpecified is not set in AssemblyInfo,
+        /// it defaults to being true (which means assembly is not removed).
+        /// That is important for compatibility of old ApiPort tool with the new service, after 
```

Comment:
Using the term *new service* won't make sense in the very near future

Reply:
Fixed comment.